app/models/manageiq/providers/openshift/inventory/collector/target_collection.rb

```diff
@@ -0,0 +1,15 @@
+class ManageIQ::Providers::Openshift::Inventory::Collector::TargetCollection < ManageIQ::Providers::Openshift::Inventory::Collector
+  include ManageIQ::Providers::Kubernetes::ContainerManager::TargetCollectionMixin
+
+  def inventory(entities)
+    full_inventory = clean_inventory(entities)
+
+    # Fill pods from Targets
+    full_inventory['pod'] = pods
+    # Fill pods references
+    full_inventory.merge!(pods_references(pods))
+
+    # TODO(lsmola) I am collecting just namespaces, there are OSE projects, but return [], should it work?
```

Comment:
@cben not sure why asking oapi for projects return [], should there be any?

Reply:
@Ladas I get the projects using `curl -k -s     -H "Authorization: Bearer $OSH_TOKEN"     https://$OSH_HOST:8443/oapi/v1/projects`

See also discovery
`curl -k -s     -H "Authorization: Bearer $OSH_TOKEN"     https://$OSH_HOST:8443/oapi/v1|grep projects -B1 -A12`

Is it possible you are using the kubernetes API for the openshift entities?
https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/135/files#diff-7f4df1e603392ee46ca753872226a938R30

Reply:
ok, I'll try again with followup PR. I am pretty sure I was using the same code we use in full refresh, that should be talking to the right endpoint.